src/backend/distributed/operations/repair_shards.c

```diff
@@ -158,6 +178,125 @@ master_copy_shard_placement(PG_FUNCTION_ARGS)
 }
 
 
+/*
+ * ColocationSizeInBytes returns the size in bytes of a set of colocated tables.
```

Comment:
Why the name starts with `colocation`? We give a list of shards and it calculates the shards?

Reply:
I am changing it as `ShardListSizeInBytes`